src/components/Philosophy.jsx

```diff
@@ -0,0 +1,27 @@
+
+import './Reviews.css';
+
+export const Philosophy = ({ articles }) => {
+	return (
+		<>
+			<a className='cursor-pointer' href="#">
+				<div id="Philosophy" className="font-inter wrapper relative text-darkblue bg-yogawhite flex py-[65px] mx-auto lg:py-[109px] pl-[5px] md:pl-[10px]">
+					<div id='slider' className='mx-[4.5px] w-full h-full overflow-x-scroll whitespace-nowrap scroll smooth xl:overflow-x-hidden'>
```

Comment:
it worked for you so flawless. We really had a struggle to do the side scroll. Good job!